sites/geohub/src/components/util/MiniMap.svelte

```diff
@@ -24,6 +24,9 @@
 	export let isLoadMap = false;
 	export let layer: VectorLayerTileStatLayer = undefined;
 	export let band: string = undefined;
+	export let tool: string = undefined;
```

Comment:
I think you decided not to change MiniMap (preview) for tools now. Are these changes necessary for this PR? could you revert them to original one?